### PR TITLE
Wykonane zadania 1.2-1.5

### PR DESCRIPTION
**Pull request by Miretpl**

### CODE REVIEW

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 1]
```diff
@@ -50,4 +41,15 @@ public int pop() {
 		return ret;
 	}
 
+}
+
+class Node {
+
+	final int value;
+	Node prev, next;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: What about encapsulation task?

File: doc/allclasses-index.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE HTML>
```
Comment (kowallus) [inline doc/allclasses-index.html]: javadocs can be generated out of source. Only source should be commited.

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 
+
+// TODO: class is not used. Consider deleting it.
 public class IntLinkedList {
 
-	Node last;
+	private static final int IF_EMPTY = -1;
+	private Node last;
+  
 	int i;
```
Comment (kowallus) [inline src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: Unused private field could be removed.

Comment (Miretpl) [reply src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: Int i field removed.

File: doc/allclasses-index.html (hunk 2 above)
Comment (Miretpl) [reply doc/allclasses-index.html]: Doc folder removed from repository.

File: src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java (hunk 1 above)
Comment (Miretpl) [reply src/main/java/edu/kis/vh/nursery/list/IntLinkedList.java]: Created setters and getters for fields in Node class.

**Comment by kowallus** [comment]
OK

**Pull request merged**